lib/analyze/collect/metadata.js

```diff
@@ -208,6 +208,22 @@ function extractPublisher(packageJson, maintainers) {
 }
 
 /**
+ * Extract the scope from the package name.
+ *
+ * @param {string} pkgName the name of the package
```

Comment:
Can we receive the whole `packageJson` here to be consistent with other extractors on the same file?